setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='cronitor',
-    version='4.3.0',
+    version='4.3.1',
```

Comment:
i say we at least bump at least the minor version num. this is kinda a big deal ;)

Reply:
We can! I meant to bump it minor haha, was a mistake. It's backwards compatible so I thought could be minor